### ISSUE TITLE
Windows: msvc_runtime failing to install

### ISSUE THREAD

**Issue by caniko** [DeepLabCut/DeepLabCut#260]
**Your Operating system and DeepLabCut version**
Windows 10; Anaconda Env; dlc 2.x

Please state your operating system, env, and which version of DeepLabCut you are using.
Example: Ubuntu 16.04 LTS, with an Anaconda Env, & DeepLabCut1.x or 2.x.
 
**Describe the bug**
When I execute the provided dlc-windowsGPU.yaml file from the repository, I get the following console log from conda (including the error):
```
WARNING: The conda.compat module is deprecated and will be removed in a future release.
Warning: you have pip-installed dependencies in your environment file, but you do not list pip itself as one of your conda dependencies.  Conda may not use the correct pip to install your packages, and they may end up in the wrong place.  Please add an explicit pip dependency.  I'm adding one for you, but still nagging you.
Collecting package metadata: done
Solving environment: done
Preparing transaction: done
Verifying transaction: done
Executing transaction: failed
ERROR conda.core.link:_execute(568): An error occurred while installing package 'defaults::msvc_runtime-1.0.1-vc14_0'.
PermissionError(13, 'Permission denied')
Attempting to roll back.

Rolling back transaction: done

[Errno 13] Permission denied: 'C:\\Users\\Can\\Miniconda3\\envs\\dlc\\vcruntime140.dll'
```

**To Reproduce**
Follow the README.md file on this page: https://github.com/AlexEMG/DeepLabCut/tree/master/conda-environments

**Expected behavior**
Error message in the log above.

**Additional information**
I tried installing both through elevated Anaconda prompt and without, and it doesn't change the outcome.
As expected, adding pip to the dependency list doesn't resolve the issue

**Comment by MMathisLab** [comment]
https://stackoverflow.com/questions/44044298/permissionerror-errno-13-permission-denied-c-program-files-python35-lib

**Comment by caniko** [comment]
> 
> 
> https://stackoverflow.com/questions/44044298/permissionerror-errno-13-permission-denied-c-program-files-python35-lib

I was just editing my post:

> I tried installing both through elevated Anaconda prompt and without, and it doesn't change the outcome.

**Comment by MMathisLab** [comment]
and you ran as an admin? Do you have admin rights on the computer?

 You can also delete that line from the conda file - there was a anaconda/python issue mid-feb and this was the fix, but it's not typically required

**Comment by caniko** [comment]
> 
> 
> and you ran as an admin? Do you have admin rights on the computer?
> 
> You can also delete that line from the conda file - there was a anaconda/python issue mid-feb and this was the fix, but it's not typically required

Yes, I have admin rights on the respective computer. I get the error both on elevation and non-elevation. By elevation I mean that I launch anaconda as administrator.

**Clearification**
```
name: dlc
dependencies:
  - python=3.6
  - pip
  - msvc_runtime <--------(THIS LINE?)
  - tensorflow-gpu==1.12
  - cudnn=7
  - wxpython
  - jupyter
  - pytables==3.4.4
  - pip:
    - deeplabcut
```

**Comment by MMathisLab** [comment]
okay great, yep! just delete the`` - msvc_runtime`` <--------(THIS LINE?)

**Comment by caniko** [comment]
> 
> 
> okay great, yep! just delete the` - msvc_runtime` <--------(THIS LINE?)

Did what we agreed uppon, and got the same error again:
```
ERROR conda.core.link:_execute(568): An error occurred while installing package 'defaults::vs2015_runtime-14.15.26706-h3a45250_0'.
PermissionError(13, 'Permission denied')
Attempting to roll back.

Rolling back transaction: done

[Errno 13] Permission denied: 'C:\\Users\\Can\\Miniconda3\\envs\\dlc\\vcruntime140.dll'
```

**Comment by MMathisLab** [comment]
hmm... then it seems you don't have permissions to edit the .dll files - are you sure the user (``Can``) on Windows has admin rights?

**Comment by caniko** [comment]
Yes. It is my, Can's, own personal computer

**Comment by MMathisLab** [comment]
do you have visual studio installed?

**Comment by caniko** [comment]
Yes

The following parts are installed:
![image](https://user-images.githubusercontent.com/29519599/55890330-cb9ed080-5bb2-11e9-8c19-826de2c99f31.png)

**Comment by MMathisLab** [comment]
okay, interesting. It seems MSVC is installed then, so I'm really not sure why you get the permission issues.

**Comment by caniko** [comment]
It works now!

**Comment by MMathisLab** [comment]
great, what was the issue (in case others fall upon it!)

**Comment by dbabbitt** [comment]
I can get it to work by opening a new Administrator: Anaconda Prompt and get it not to work by running it again in the same prompt.

**Comment by jedisom** [comment]
In case anyone else stumbles upon this issue, I was having similar errors even when running in Administrator mode.  I found that somehow, when I was removing my environment with `conda remove --name <env_name_here> --all` the process didn't run correctly and got corrupted.  Due to this, there were still files in the miniconda3/envs folder for this environment.  I manually deleted the environment folder (<env_name_here>) and re-ran by `conda env create` command and it worked.
